presto/presto.sh

```diff
@@ -55,8 +50,8 @@ function wait_for_presto_cluster_ready() {
 
 function get_presto(){
   # Download and unpack Presto server
-  wget https://repo1.maven.org/maven2/com/facebook/presto/presto-server/${PRESTO_VERSION}/presto-server-${PRESTO_VERSION}.tar.gz
-  tar -zxvf presto-server-${PRESTO_VERSION}.tar.gz
+  wget https://storage.googleapis.com/starburstdata/presto/${PRESTO_MAJOR_VERSION}e/${STARBURST_PRESTO_VERSION}/presto-server-${STARBURST_PRESTO_VERSION}.tar.gz
```

Comment:
Use `-O presto-server-${STARBURST_PRESTO_VERSION}.tar.gz` or `-O presto-server.tar.gz` to enforce specific file name. Don't rely on `wget`'s picking up the output file name.

Reply:
Also, per bash code style, every variable should be in quotes, just in case there is some whitespace.

```suggestion
  wget "https://storage.googleapis.com/starburstdata/presto/${PRESTO_MAJOR_VERSION}e/${STARBURST_PRESTO_VERSION}/presto-server-${STARBURST_PRESTO_VERSION}.tar.gz" -O "...."
```